sequence_processing_pipeline/FastQCJob.py

```diff
@@ -83,6 +83,9 @@ def _find_projects(self, path_to_run_id_data_fastq_dir, is_raw_input):
             # extract only fastq files from the list
             files = [x for x in files if x.endswith('.fastq.gz')]
 
+            # remove all files in 'zero_files' sub-folder
+            files = [x for x in files if 'zero_files' not in x]
```

Comment:
Out of curiosity, why split in two vs. just doing in one loop?

Reply:
mainly for readability. Let me see if I can compound the if clause - don't think I've ever tried to use and inside a list comprehension before.